### PR TITLE
chore(docs): docs release notes  November 22-28, 2024 - first draft

### PR DESCRIPTION
**Pull request by sujitnewrelic**
Release note draft November 22-28, 2024.

### CODE REVIEW

File: src/content/docs/release-notes/docs-release-notes/docs-11-29-2024.mdx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+version: 'version: November 22-28, 2024'
+---
+
+### New doc
+
+* [Snowflake integration with flex](/docs/infrastructure/host-integrations/host-integrations-list/snowflake-integration) describes how to install Snowflake dashboards and view your Snowflake data in New Relic.
+
+### Major change
+
+* Updated the [Certifications, standards, and regulations](/docs/security/security-privacy/compliance/regulatory-audits-new-relic-services) information for New Relic and added details on [PCI DSS](/docs/security/security-privacy/compliance/certificates-standards-regulations/pci-dss).
```
Comment (ubanerjeeNR) [inline src/content/docs/release-notes/docs-release-notes/docs-11-29-2024.mdx:13]: what do you mean by added details on PCI DSS? Added what details? If PCI DSS is the name of the topic to which the details are added, consider rewording it as follows:

Updated the [Certifications, standards, and regulations](/docs/security/security-privacy/compliance/regulatory-audits-new-relic-services) information for New Relic, and added A_B_C details to [PCI DSS](/docs/security/security-privacy/compliance/certificates-standards-regulations/pci-dss).

-----------

Replace A_B_C with the actual details added.

Comment (sujitnewrelic) [reply src/content/docs/release-notes/docs-release-notes/docs-11-29-2024.mdx]: PCI DSS is a new page. So, moved it to "New Doc".

**Review by ubanerjeeNR — DISMISSED** [review]
LGTM

**Review by WriteMayur — APPROVED** [review]
looks good

**Pull request merged**